### ISSUE TITLE
Twilight Forest Upgrade Results in Tick Damage / Fire / Slowness

### ISSUE THREAD

**Issue by Hexadiagonal** [NillerMedDild/Enigmatica2#717]
**Modpack version**
Input modpack version here 1.77 Upgrade to 1.78

**Issue**

Pre-existing portals to safe regions of the Twilight Forest are now resulting in ~~Biome-progression related damage~~ burning damage and the Slow effect . Deleting DIM7 does not appear to change the damage and experience as per a normal reset. Advancement resets also do not help.

**Logs**
Input pastebin link to crash-report or latest log, if applicable

NA

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by NillerMedDild** [comment]
Ah I see you posted it on their issue tracker as well, thanks :D
https://github.com/TeamTwilight/twilightforest/issues/1020

**Comment by TrapStoner** [comment]
The problem is from Advanced Rocketry. When i removed it i no longer was burning and having slowness in the twilight forest. Trying to see if there is a workaround now.

**Comment by Hexadiagonal** [comment]
> The problem is from Advanced Rocketry. When i removed it i no longer was burning and having slowness in the twilight forest. Trying to see if there is a workaround now.

Updated the original post to reflect general Burning damage and the Slow effect, instead of the Biome reference.

**Comment by Hexadiagonal** [comment]
On reviewing a historical issue [here](https://github.com/zmaster587/AdvancedRocketry/issues/933) I checked up on /word/advRocketry. I tried removing the PlanetDefs.xml and temp.dat files to see if it would reset.

On regeneration, it believes that the Twilight Forest is about 50% hotter than its internal representation of Mercury.

DIMID 0 [Overworld]
<avgTemperature>287</avgTemperature>

DIMID 101 [Mercury]
<avgTemperature>455</avgTemperature>

DIMID 7 [Twilight]
<avgTemperature>743</avgTemperature>

Interestingly, the Slow effect appears to be gone. I had looked at this before and it was also in the 500s. Is there some value that's getting cached and re-added upon itself each regen?

**Comment by TrapStoner** [comment]
Just bring the temp below 450 and you will be good to go. Slowness and fire are gone!

**Comment by NillerMedDild** [comment]
Awesome, thanks!

**Comment by Hexadiagonal** [comment]
@NillerMedDild I successfully performed the following steps:

1. Open advancedRocketry.cfg
2. Change resetPlanetsFromXML to 'true'
3.  Open enigmatica-2/config/advRocketry/planetDefs.xml
4. Changed the Twilight entry for DIMID="7" orbitalDistance 200
4.. Delete world/advRocketry/planetDefs.xml & world/advRocketry/temp.dat
5.  Load game and drop into the Twilight Forest dimension.

I did not encounter additional issues. It seems the star that the Twilight Forest dimension orbits is at a heat of 200, and the Twilight Forest dimension is close to the star (previously 60, if I recall, compared to an 'earthlike' of 100).

I'm guessing download of the modpack brings in advRocketry's internal representation of things; I don't know where it's stored, because even the **extra** planetDefs.xml and temp.data get regenned if you don't also ensure that the config entry of resetPlanetsFromXml is true.

I'm going to go ensure that my entry is closed on the Twilight Forest end, but I'm good to go over here.
